baseplate/config.py

```diff
@@ -167,6 +167,24 @@ def Base64(text):
         raise ValueError(*exc.args)
 
 
+def File(mode="r"):
+    """A path to a file.
+
+    This takes a path to a file and returns an open file object, like
```

Comment:
I've added some example usage for the file thingy. The general goal is to fail fast if any config is bad.